install

```diff
@@ -0,0 +1,208 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# Reset
+Color_Off='\033[0m' # Text Reset
+
+# Regular Colors
+Red='\033[0;31m'   # Red
+Green='\033[0;32m' # Green
+Dim='\033[0;2m'    # White
+
+# Bold
+Bold_Green='\033[1;32m' # Bold Green
+Bold_White='\033[1m'    # Bold White
+
+error() {
+    echo -e "${Red}error${Color_Off}:" "$@" >&2
+    exit 1
+}
+
+info() {
+    echo -e "${Dim}$@ ${Color_Off}"
+}
+
+info_bold() {
+    echo -e "${Bold_White}$@ ${Color_Off}"
+}
+
+success() {
+    echo -e "${Green}$@ ${Color_Off}"
+}
+
+tildify() {
+    if [[ $1 = $HOME/* ]]; then
+        local replacement=\~/
+
+        echo "${1/$HOME\//$replacement}"
+    else
+        echo "$1"
+    fi
+}
+
+case $(uname -ms) in
+'Darwin x86_64')
+    target=x86_64-apple-darwin
+    ;;
+'Darwin arm64')
+    target=aarch64-apple-darwin
+    ;;
+'Linux x86_64' | *)
+    target=x86_64-unknown-linux-musl
+    ;;
+esac
+
+today=$(date +"%Y-%m-%d")
+dune_uri="https://download.dune.ci.dev/$today/$target/dune"
+
+install_dir=$HOME/.dune
+bin_dir=$install_dir/bin
+exe=$bin_dir/dune
+tilde_bin_dir=$(tildify $bin_dir)
+
+if [[ ! -d $bin_dir ]]; then
+    mkdir -p "$bin_dir" ||
+        error "Failed to create install directory \"$bin_dir\""
+fi
+
+curl --fail --location --progress-bar --output "$exe" "$dune_uri" ||
+    error "Failed to download dune from \"$dune_uri\""
+
+chmod +x "$exe" ||
+    error 'Failed to set permissions on dune executable'
+
+
+
+success "dune was installed successfully to $Bold_Green$(tildify "$exe")"
+
+echo
+
+case $(basename "$SHELL") in
```

Comment:
Instead of directly exporting, what do you think about asking the user? It would prevent people that tries to install the newer version to have the export multiple times and also change the location for the people who want. WDYT?